docs/cimpl6ToolingReference.md

```diff
@@ -342,83 +343,69 @@ The Content Profile (CP) specifies the profiles to be included in the IG. As exp
+As illustrated above, the FHIR IG may include models from multiple namespaces. The first job of the CP is to specify the profiles that should be presented in the IG, out of the available `Entry` models in class files in the _spec_ directory.
 
-The syntax of a Content Profile file is:
+To include the profile of an `Entry` in the IG, list the class name in the CP. The list must be arranged by namespace, as shown in [the example below](#content-profile-example). To include all Entries in a namespace, mark the namespace with a wildcard `*`. We refer to any `Entry` explicitly listed for inclusion as "primary" profiles.
 
-```
-Grammar:  ContentProfile 1.0
-
-Namespace:  <namespace-1>
-    <Entry-1>:
-        <Element-1> MS
-        <Element-2> MS
-        ...
-    <Entry-2>: NP
-    ...
-Namespace:  <namespace-2>
-    <Entry-1>:
-        <Element-1> MS
-        <Element-2> MS
-        ...
-    <Entry-2>: NP
-    ...
-```
+> **Note**: Only `Entries` result in FHIR profiles. `Groups` and `Elements` result in extensions. Inclusion of extensions in the IG is automatic, and cannot be specified in the CP.
 
-For example:
+#### Specifying No-Profile Classes
 
-```
-    Grammar: ContentProfile 1.0
+When you specify an `Entry` to be included in an IG, it may indirectly pull in other `Entries`. For example, an Observation references other classes including Patient, Specimen, Device, and Practitioner. In turn, Specimen could reference a Procedure, Device an Organization, etc, until a large number of classes are involved. By default, if there is a CIMPL model for a referenced class, SHR-CLI will create a profile for that class, and include it in the IG. We refer to these dependent profiles as _supporting_ profiles.
 
-    Namespace: obf
-        Patient:
-            BirthDate MS
-        Encounter: NP
-        EpisodeOfCare: NP
+The No-Profile (`NP`) flag is a way to put boundaries around your IG, and eliminate some or all supporting profiles. The No-Profile (`NP`) flag after a class instructs SHR-CLI to _not_ profile a class, and therefore, exclude it from the list of profiles in the IG. To exclude all classes in a namespace, put the `NP` flag after the namespace name.
 
-    Namespace: vital
-    BloodPressure:
-        Components.ObservationComponent MS
-        RelevantTime MS
-    BodyHeight:
-        DataValue MS
-        RelevantTime MS
-    BodyWeight:
-        DataValue MS
-        RelevantTime MS
-        ...
-```
+The `NP` flag can be applied to both `Entries` and `Groups`. An example of applying `NP` to a `Group`, if there is a model of Address that is different than FHIR's Address data type, then specifying `Address: NP` will use FHIR's Address rather than a profiled version of Address.
```

Comment:
Clarification:  the note above mentions that `Group` creates extensions, in the case of the `NP` flag applied to a group, this means that an extension is not created?

Reply:
Don't know what you mean.